.travis.yml

```diff
@@ -32,20 +32,23 @@ jobs:
 
   include:
     # runs linting and tests with current locked deps
-
     - stage: "Tests"
       name: "Tests"
       script:
-        - yarn lint:hbs
-        - yarn lint:js
-        - yarn test
+        - yarn lint
+        - yarn test:ember
```

Comment:
- The initial npm-scripts `lint:hbs` and `lint:js` can be now called with the single npm-script `lint`
- The initial npm-script `test` used to trigger `ember test`, we now have to call the npm-script `test:ember`

Also this is what is used in a fresh Ember add-on v3.24.

**`package.json` in Ember v3.24**

https://github.com/ember-cli/ember-addon-output/blob/07fe28b1c23a909ce9dabd04d482003c56614429/package.json#L15-L27

**`.travis.yml` in Ember v3.24**

https://github.com/ember-cli/ember-addon-output/blob/07fe28b1c23a909ce9dabd04d482003c56614429/.travis.yml#L38-L39